server/test/index.test.ts

```diff
@@ -45,28 +47,32 @@ describe("http requests testing", () => {
   });
 
   test("GET request of second note by id", async () => {
-    const response = await request(app).get("/notes/2");
+    const response = await request(app).get(`/notes/${secondId}`);
     expect(response.statusCode).toStrictEqual(200);
     const noteJson: Note = response.body;
     expect(noteJson.title).toBe("2nd");
     expect(noteJson.noteContent).toBe("This is Second note");
   });
 
-  test("GET request by id to get 404 response", async () => {
-    const response = await request(app).get("/notes/99");
+  test("GET request by a random id to get 404 response", async () => {
+    const response = await request(app).get(
+      "/notes/c61f1fc9-e0d2-43ce-a6c1-b5436cdebe7d"
```

Comment:
Where is `c61f1fc9-e0d2-43ce-a6c1-b5436cdebe7d` coming from

Reply:
It's just an example in uuid format as `99` is invalid now.